cmd/skaffold/app/cmd/flags.go

```diff
@@ -0,0 +1,264 @@
+		DefValue:      false,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug"},
+	},
+	{
+		Name:          "cache-file",
+		Usage:         "Specify the location of the cache file (default $HOME/.skaffold/cache)",
+		Value:         &opts.CacheFile,
+		DefValue:      "",
+		FlagAddMethod: "StringVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug"},
+	},
+	{
+		Name:          "insecure-registry",
+		Usage:         "Target registries for built images which are not secure",
+		Value:         &opts.InsecureRegistries,
+		DefValue:      []string{},
+		FlagAddMethod: "StringSliceVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug"},
+	},
+	{
+		Name:          "enable-rpc",
+		Usage:         "Enable gRPC for exposing Skaffold events (true by default for `skaffold dev`)",
+		Value:         &opts.EnableRPC,
+		DefValue:      false,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug", "deploy"},
+	},
+	{
+		Name:          "rpc-port",
+		Usage:         "tcp port to expose event API",
+		Value:         &opts.RPCPort,
+		DefValue:      constants.DefaultRPCPort,
+		FlagAddMethod: "IntVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug", "deploy"},
+	},
+	{
+		Name:          "rpc-http-port",
+		Usage:         "tcp port to expose event REST API over HTTP",
+		Value:         &opts.RPCHTTPPort,
+		DefValue:      constants.DefaultRPCHTTPPort,
+		FlagAddMethod: "IntVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug", "deploy"},
+	},
+	{
+		Name:          "label",
+		Shorthand:     "l",
+		Usage:         "Add custom labels to deployed objects. Set multiple times for multiple labels",
+		Value:         &opts.CustomLabels,
+		DefValue:      []string{},
+		FlagAddMethod: "StringSliceVar",
+		DefinedOn:     []string{"dev", "run", "debug", "deploy"},
+	},
+	{
+		Name:          "toot",
+		Usage:         "Emit a terminal beep after the deploy is complete",
+		Value:         &opts.Notification,
+		DefValue:      false,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"dev", "build", "run", "debug", "deploy"},
+	},
+	{
+		Name:          "tail",
+		Usage:         "Stream logs from deployed objects",
+		Value:         &opts.Tail,
+		DefValue:      false,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"deploy"},
+	},
+	{
+		Name:          "force",
+		Usage:         "Recreate kubernetes resources if necessary for deployment, warning: might cause downtime!)",
+		Value:         &opts.Force,
+		DefValue:      false,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"deploy"},
+	},
+	{
+		Name:          "tail",
+		Usage:         "Stream logs from deployed objects",
+		Value:         &opts.TailDev,
+		DefValue:      true,
+		FlagAddMethod: "BoolVar",
+		DefinedOn:     []string{"dev", "run", "debug"},
+	},
+	{
+		Name:          "force",
+		Usage:         "Recreate kubernetes resources if necessary for deployment, warning: might cause downtime!)",
```

Comment:
```suggestion
		Usage:         "Recreate kubernetes resources if necessary for deployment (warning: might cause downtime!)",
```

Reply:
can we move the two "force"-s next to each other with a comment that yes, this is intentional?

Reply:
done! added a commit